src/js/background.js

```diff
@@ -533,6 +533,7 @@ Badger.prototype = {
       Migrations.migrateDntRecheckTimes,
       // Need to run this migration again for everyone to #1181
       Migrations.migrateDntRecheckTimes2,
+      Migrations.migrateMDFPAwareStorageObjects,
```

Comment:
Ah, the tests are failing because Badger fails to initialize properly since there is no such migration as `migrateMDFPAwareStorageObjects`:

![Screenshot from 2019-06-17 14:04:46](https://user-images.githubusercontent.com/794578/59626074-e2105d80-9108-11e9-85ac-85454514db00.png)

Reply:
Our migrations system doesn't support anything except adding a new migration to the back of this array. If you add a migration to the middle of the array, that migration will not run for existing Badgers (as it will be below their migration level); the migration at the back of the array will be re-run instead.

Reply:
Thanks for pointing that out! I'll move it to the end of the array.